hummingbot/market/bitfinex/bitfinex_auth.py

```diff
@@ -40,7 +40,7 @@ def generate_api_headers(self, path, body):
 
     # private methods
     def _make_nonce(self) -> int:
-        nonce = int(round(time.time() * 1000000))
+        nonce = int(round(time.time() * 1_000_000))
```

Comment:
I am wondering if this is the ideal solution for this. From my understanding of the issues mentioned in previous comments, this has to do with the time in which BitFinex receives the requests and simply multiplying the `nonce` with a larger number should not change this behaviour. 🤔